### ISSUE TITLE
[Bug]: Can't directly place Create's Funnels into Drawers

### ISSUE THREAD

**Issue by SeymourGx** [MattiDragon/ExtendedDrawers#46]
### Description of the issue

Attempting to store an Andesite Funnel or Brass Funnel from Create directly within a Drawer will cause the Funnel to externally attach to the Drawer. (When using default keybinds,) both Right-Clicking and Shift+Right-Clicking will fail to store the funnel. This also occurs on Drawer Access Points. Maybe this is intended, but switching to this from Storage Drawers makes me unsure as that mod allowed you to Right-Click to store the Funnel and Shift+Right-Click to attach it externally -- additionally strange is that Hoppers, Chutes, and other item-transporters can be manually placed in Drawers normally but Funnels specifically do not. I imagine this is simply how Create handles Funnels attaching to any Fabric Storage block, and would likely need a special compatibility tweak to deal with. Making this a Feature Request? Unsure if intended functionality so leaving as a bug for now.

### Reproduction steps

1. Approach a Drawer of any sort.
2. Attempt to directly place a Funnel in the Drawer by Right-Clicking, Shift+Right-Clicking, etc.
3. Fail.
![ExtendedDrawersBug](https://user-images.githubusercontent.com/51757397/222885446-727110f1-7174-465d-8574-b2ce43280a58.gif)

### Mod version

1.3.6

### Operating System

Windows 11

### Minecraft version

1.19.2

### Other mods

Fabric API: 0.73.2
Create Fabric: 0.5.0.i-961

### Server

Happens

### Client

Happens

### Logs

N/A

### Extra Context

Note: It IS possible to use indirect methods to store the Funnels into a Drawer, such as a Hopper, Conveyor Belt and Funnel (ironically), etc. This bug report is meant to specifically address placing Funnels into Drawers directly, i.e., using the Player and no external mechanisms. I file this report essentially so I don't have to slap a Hopper on my Drawer Access Point and mar the aesthetics/functionality just for Funnels.

**Comment by MattiDragon** [comment]
I'll have to look into this, but I suspect create is handling the placement of funnels weirdly

**Comment by IzMichael** [comment]
Yeah, Create allows funnels to be placed on all blocks, without requiring a shift-click

**Comment by nicolas-rabault** [comment]
To work around this one, I use a hopper or something similar connected to the drawer and drop the funnels on it.
